src/index.js

```diff
@@ -4,4 +4,5 @@ import ReactDOM from 'react-dom';
 import App from 'components/App';
 import 'index.css';
 
+/* eslint-disable-next-line react/jsx-filename-extension */
```

Comment:
Maybe a comment here to explain what this is doing? I'm not really sure what's going on here

Reply:
The rule requires that JSX is only allowed in .jsx files. I initially assumed that React looks specifically for index.js as the entrypoint, but it appears to work when I rename it to index.jsx, so i'll just do that.